server/routes/boards-routes.js

```diff
@@ -32,7 +33,7 @@ router.post('/', async (req, res, next) => {
 
   let savedLifecycles;
   try {
-    savedLifecycles = await Lifecycle.insertMany(lifeCycles);
+    savedLifecycles = await Lifecycle.insertMany(lifecycles);
   } catch (e) {
```

Comment:
We also have to add members into the Board Schema along with their role.
The Person who is creating the board will be SUPERADMIN

Reply:
I've already added that, see line no 26-32 of board-routes.js

Reply:
Sorry, my bad!